src/components/web-view.js

```diff
@@ -1,15 +1,25 @@
-import React from "react";
+import React from 'react';
+import PropTypes from 'prop-types';
+import mapProperties from '../properties-mapping.js';
 import { UiNode } from './types/ui-node.js';
-import mapProperties from "../properties-mapping.js";
+import { TypeValidator } from './util/type-validators.js';
+import { WebViewAction } from './types/enums.js';
 
 export function WebView(props) {
-  return React.createElement("webView", mapProperties("WebView", props));
+  return React.createElement('webView', mapProperties('WebView', props));
 }
 
 WebView.propTypes = {
-  ...UiNode.PropTypes
+  ...UiNode.PropTypes,
+
+  // Properties
+  action: PropTypes.oneOf(Object.values(WebViewAction)),
```

Comment:
how is `action` supposed to be used?  We're not using it the Catalog app.  I'm not sure if it makes sense.

Reply:
Should we remove `action` from here and from the documentation?

Reply:
I think we should remove it